README.md

```diff
@@ -173,6 +173,11 @@ component itself.
 </div>
 ```
 
+You can also use the `styles` attributes to make the "Shadow DOM" inherits the global CSS:
```

Comment:
Do you think a comment is needed in the inline styles section of the README about inline styles, or do you think it's safe to remove that section altogether?